src/constructors.jl

```diff
@@ -103,3 +103,24 @@ end
 function Base.call{T, N}(::Type{NullableArray{T, N}})
     NullableArray(T, ntuple(i->0, N))
 end
+
+#----- Conversion from Array of Nullables #1 -----------------------------------#
+# Take an Array of Nullable objects and convert it into a NullableArray
+function Base.convert{S,T,N}(::Type{NullableArray{S,N}},in::Array{Nullable{T},N})
+   out = NullableArray{S,N}(Array(S,size(in)),Array(Bool,size(in)))
+   for i = 1:length(in)
+       (out.isnull[i] = in[i].isnull) ? nothing : out.values[i] = in[i].value
```

Comment:
You mean that `get` gives slower code when benchmarking it? I would have thought it would be inlined and that the redundant checks for nullity would go away, but maybe that's too optimistic.